src/simulations/local_elec.py

```diff
@@ -556,7 +558,24 @@ def policy_to_rewardvar(
             bool_penalty, date, loads, E_req_only, h, last_step, home_vars)
 
         if self.prm['grd']['manage_voltage']:
-            hourly_line_losses, voltage = self.network.pf_simulation(home_vars['netp'])
+            # p_non_flex, _, _ = self._get_passive_vars(h)
+            p_non_flex = 0
+            q_heat_home_car_non_flex = p_non_flex \
+                * math.tan(math.acos(self.prm['grd']['pf_non_flex_heat_home_car']))
+            q_heat_home_flex = home_vars['tot_cons'] \
+                * math.tan(math.acos(self.prm['grd']['pf_flexible_heat_home']))
```

Comment:
could you put this in a method in env and then use it here and in explorer? rather than repeating the same thing twice, this is more likely to cause issues in the future if we have to edit things twice

Reply:
Good point, see `_calculate_reactive_power`

Reply:
great thanks!